src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBMetricsProvider.cs

```diff
@@ -93,45 +79,6 @@ public async Task<CosmosDBTriggerMetrics> GetMetricsAsync()
                 partitionCount = 1;
                 remainingWork = 1;
             }
-            catch (Exception e) when (e is CosmosException || e is InvalidOperationException)
-            {
-                if (!TryHandleCosmosException(e))
```

Comment:
Yes, it makes sense to me to remove this exception swallowing. It looks like a lot of effort was put into this code when it was originally added by @ealsur. It'd be good to get his signoff on this as well.

From the links you shared Alexey, we have upstream handling already for any exceptions coming from this code on the scale monitor path. My primary concern with this was whether letting these through was going to start failing scenarios that are currently passing (i.e. regression).